src/index.ts

```diff
@@ -263,7 +271,14 @@ class VirtualModulesPlugin {
       const fts = compiler.fileTimestamps as any;
       if (virtualFiles && fts && typeof fts.set === 'function') {
         Object.keys(virtualFiles).forEach((file) => {
-          fts.set(file, +virtualFiles[file].stats.mtime);
+          const mtime = +virtualFiles[file].stats.mtime;
```

Comment:
The `+` in `+virtualFiles[file].stats.mtime` apparently casts "mtime" into a string. Savvy trick, didn't know that though `Number()` probably would've been clearer.